smart_open/smart_open_lib.py

```diff
@@ -57,6 +57,7 @@
 from smart_open import doctools
 
 logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
Better add NullHandler in `__init__.py`, to avoid having to add it to each logger individually?